<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>353</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>协议解析</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lbl_protocalType">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>协议类型：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_protocalData">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>协议数据：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtn_Analysis">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>278</y>
      <width>141</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>解    析</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbx_protocalType">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>35</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>1826F456(CHM)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>182756F4(BHM)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1801F456(CRM)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BRM多包数据报文</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BCP多包数据报文</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1808F456(CML)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100956F4(BRO)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100AF456(CRO)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>181056F4(BCL)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BCS多包数据报文</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1812F456(CCS)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>181356F4(BSM)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BMV多包数据报文</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BMT多包数据报文</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>101956F4(BST)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>101AF456(CST)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>181C56F4(BSD)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>181DF456(CSD)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>081E56F4(BEM)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>081FF456(CEM)</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="txtEdt_protocalData">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>78</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>20</width>
      <height>321</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>278</y>
      <width>141</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>清    空</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtEdt_result">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>39</y>
      <width>301</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>15</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>解析结果：</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
